src/test/java/com/google/devtools/build/lib/bazel/repository/downloader/HttpStreamTest.java

```diff
@@ -148,10 +149,14 @@ public void smallDataWithValidChecksum_timesOutInCreateRepeatedly() throws Excep
     when(connection.getHeaderField("Accept-Ranges")).thenReturn("bytes");
     thrown.expect(SocketTimeoutException.class);
 
+    Field field = RetryingInputStream.class.getDeclaredField("MAX_RESUMES");
+    field.setAccessible(true);
+    int maxResumes = (int) field.get(null);
```

Comment:
oh! i assumed that MAX_RESUMES was a public field :)
we can just go back to using regular 3

to save us the turn-around time (this pull request has been out for quite some time already!) i'll make the adjustment on our end :)